filament/src/ShadowMap.cpp

```diff
@@ -60,6 +60,9 @@ ShadowMap::ShadowMap(FEngine& engine) noexcept :
     }
 }
 
+void ShadowMap::terminate(FEngine& engine) {
```

Comment:
Is this necessary?

Reply:
not in this PR, I added it for consistency with other classes. It would be useful if for e.g. ShadowMap had its own UBO.